hopsworks-common/src/main/java/io/hops/hopsworks/common/util/SparkConfigurationUtil.java

```diff
@@ -161,6 +161,8 @@
             HopsUtils.IGNORE);
     addToSparkEnvironment(sparkProps, "HOPSWORKS_PROJECT_ID", Integer.toString(project.getId()),
             HopsUtils.IGNORE);
+    addToSparkEnvironment(sparkProps, "FLINK_CONF_DIR", settings.getFlinkConfDir(),
```

Comment:
Why does this need to be set for Spark jobs?

Reply:
Because we run Beam code within PySpark and I need this property in PySpark notebooks.